README.md

```diff
@@ -35,183 +27,56 @@ OpenTTD is licensed under the GNU General Public License version 2.0,
-the advantage that you only need to do this step once, rather than copying the
-data files into all OpenTTD versions you have.
+### 1.3.1) Legacy support
+Platforms, languages and compilers change.
+We'll keep support going on old platforms as long as someone is interested in supporting them, except where it means the project can't move forward to keep up with language and compiler features.
 
-Savegames, screenshots, etc are saved relative to the config file (openttd.cfg)
-currently being used. This means that if you use a config file in one of the
-shared directories, savegames will reside in the save/ directory next to the
-openttd.cfg file there.
+We guarantee that every revision of OpenTTD will be able to load savegames from every older revision (excepting where the savegame is corrupt).
+Please report a bug if you find a save that doesn't load.
 
-If you want savegames and screenshots in the directory where the OpenTTD binary
-resides, simply have your config file in that location. But if you remove this
-config file, savegames will still be in this directory (see notes in
-section 4.2 'OpenTTD directories')
+## 1.4) Installing and running OpenTTD
 
-OpenTTD comes without AIs, so if you want to play with AIs you have to download
-them. The easiest way is via the 'Check Online Content' button in the main menu.
-You can select some AIs that you think are compatible with your playing style.
-Another way is manually downloading the AIs from the forum although then you
-need to make sure that you install all the required AI libraries too; they get
-automatically selected (and downloaded) if you get the AIs via the 'Check
-Online Content'. If you do not have an AI but have configured OpenTTD to start
-an AI a message will be shown that the 'dummy' AI has been started.
+OpenTTD is usually straightforward to install, but for more help the wiki [includes an installation guide](https://wiki.openttd.org/Installation).
 
-### 4.1) (Required) 3rd party files
+OpenTTD needs some additional graphics and sound files to run.
 
-Before you run OpenTTD, you need to put the game's data files into a baseset/
-directory which can be located in various places addressed in the following
-section.
+For some platforms these will be downloaded during the installation process if required.
 
-For OpenTTD you need to acquire some third party data files. For this you have
-the choice of using the original Transport Tycoon Deluxe data files or a set
-of free data files.
+For some platforms, you will need to refer to [the installation guide](https://wiki.openttd.org/Installation)
 
-Do NOT copy files included with OpenTTD into 'shared' directories (explained in
-the following sections) as sooner or later you will run into graphical glitches
-when using other versions of the game.
 
-#### 4.1.1) Free graphics and sound files
+### 1.4.1) Free graphics and sound files
 
 The free data files, split into OpenGFX for graphics, OpenSFX for sounds and
```

Comment:
Section might want a rework - very repetitive "Open*FX" and bare links

Reply:
+1 but no ideas for solutions...